backend/datasource/query_type_mux.go

```diff
@@ -0,0 +1,135 @@
+package datasource
+
+import (
+	"context"
+	"fmt"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend/log"
+
+	"github.com/grafana/grafana-plugin-sdk-go/backend"
+)
+
+// The QueryTypeHandlerFunc type is an adapter to allow the use of
```

Comment:
I think linters will react to a documentation comment not starting with the name.
```suggestion
// QueryTypeHandlerFunc is an adapter to allow the use of
```